src/sagas/PostWork.saga.ts

```diff
@@ -0,0 +1,47 @@
+import { Frost } from '@po.et/frost-client'
+import { Actions } from 'actions'
+import { delay, SagaIterator } from 'redux-saga'
+import { call, takeLatest, put, ForkEffect } from 'redux-saga/effects'
+const { toast } = require('react-toastify')
+
+import { Configuration } from 'configuration'
+
+const FrostClient = new Frost({ host: Configuration.frostApiUrl })
+
+export const redirectToWork = (workId: string) =>
+  window.location.assign(`${Configuration.mainExplorerUrl}/works/${workId}`)
```

Comment:
See comment above. Not sure this is the best approach. Remember, it is going to take at least 10 minutes before the work is available through explorer-web.